pkgs/top-level/all-packages.nix

```diff
@@ -501,6 +501,10 @@ in
 
   iconConvTools = callPackage ../build-support/icon-conv-tools {};
 
+  validatePkgConfig = makeSetupHook
+    { deps = [ findutils pkgconfig ]; }
```

Comment:
Can you add a `name` parameter here?

I have been meaning to make that required at some point.

Reply:
Thanks! Fixed.